packages/@aws-cdk/aws-events/lib/event-bus.ts

```diff
@@ -47,6 +47,14 @@ export interface IEventBus extends IResource {
    * @param props Properties of the archive
    */
   archive(id: string, props: BaseArchiveProps): Archive;
+
+  /**
+   * Grants an IAM Principal to send custom events to the eventBus
+   * so that they can be matched to rules.
+   *
+   * @param grantee The principal (no-op if undefined)
+   */
+  grantPutEventsTo(grantee: iam.IGrantable): iam.Grant;
```

Comment:
`grantPutEvents` is already taken by the static method. Should I drop the static method and rename this ? (That would be a breaking change..)

Reply:
Resolving as this a stable module and the contributing guide says no breaking changes on stable modules.

Reply:
Yap, that is unfortunate.

Can I ask you to, in this same commit, to `@deprecate` the current `grantPutEvents` and add a new one that's called `grantAllPutEvents` as well? 

Then we are set up to rename this to `grantPutEvents` at some point in the future once we've removed deprecated APIs.

Reply:
Makes sense. It's done ! Ready for review